jiant/tasks/tasks.py

```diff
@@ -2753,33 +2747,15 @@ def load_data(self):
                     self.tokenizer_name, filename, has_labels=False
                 )
             else:
-                iters_by_split[split] = load_span_data(self.tokenizer_name, filename)
+                iters_by_split[split] = load_span_data(
+                    self.tokenizer_name, filename, label_fn=lambda x: int(x)
+                )
         self._iters_by_split = iters_by_split
 
-    def get_all_labels(self):
-        return ["False", "True"]
-
     def update_metrics(self, logits, labels, tagmask=None):
         logits, labels = logits.detach(), labels.detach()
-
-        def make_one_hot(batch, depth=2):
```

Comment:
acc_scorer basically necessitates the preds and the labels to be the same dimension. Now that the labels are integers (due to the change from multilabel -> label), we can get rid of the one hot vectorization and pass in both as a list of integers of dimension [batch_size, 1].